src/applications/vaos/tests/appointment-list/components/AppointmentsPage/index.unit.spec.jsx

```diff
@@ -108,6 +113,7 @@ describe('VAOS integration: appointment list', () => {
       'Primary care appointment',
     ]);
     expect(baseElement.querySelector('h4')).to.be.ok;
+    expect(getByText(/My reason isn’t listed/i)).to.be.ok;
```

Comment:
We have this covered in another test: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/tests/appointment-list/components/FutureAppointmentsList.va.unit.spec.jsx#L130